src/plugins/charts/public/static/components/color_picker.scss

```diff
@@ -11,6 +11,11 @@ $visColorPickerWidth: $euiSizeL * 8; // 8 columns
     transform: scale(1.4);
   }
 
+  // removes the grey background that appears on the first dot when the popover is open
+  &:focus {
+    background-color: #FFFFFF;
```

Comment:
Please use EUI variables for this. It's not pure white when using dark mode.

Reply:
Oh you are right, thanx for that @wylieconlon, I always forget the dark mode 🙂